app/components/claim/status_tag_component.rb

```diff
@@ -0,0 +1,16 @@
+class Claim::StatusTagComponent < ApplicationComponent
+  attr_reader :claim
+
+  def initialize(claim:, classes: [], html_attributes: {})
+    super(classes:, html_attributes:)
+
+    @claim = claim
+  end
+end
+
+def state_tag
+  css_class = claim.draft ? "govuk-tag--grey" : "govuk-tag--blue"
+  text = claim.draft ? I18n.t("components.state_tag_component.draft") : I18n.t("components.state_tag_component.submitted")
```

Comment:
Relative locale keys should work here :)

```suggestion
  text = claim.draft ? I18n.t(".draft") : I18n.t(".submitted")
```